pkg/timedata/timedata.go

```diff
@@ -43,9 +41,25 @@ func CalculateTimeQuantum(data [][]float64) time.Duration {
 	return 0
 }
 
-// Print out all the [timestamp,value] data provided
-func DebugLogPriceData(data [][]float64) {
+// BuildTimeSeriesLabels returns a list of short labels representing time values from the given [timestamp,value] data
+func BuildTimeSeriesLabels(data [][]float64) []string {
+	minTime := time.UnixMilli(int64(data[0][0]))
+	maxTime := time.UnixMilli(int64(data[len(data)-1][0]))
+	timeRange := maxTime.Sub(minTime)
+
+	var timeFormat string
```

Comment:
This function looks really good.

The time format will depend on the user's region; for example in US the day is after the month. Maybe there's a library out there to convert to user's local format? 

The locale language to use can also be determined by host local environment variables. E.g:

![DeepinScreenshot_select-area_20210927112336](https://user-images.githubusercontent.com/168240/134964989-38b784a9-5594-4183-ba6f-7c4bbd31f561.png)

For example, in [`pkg/humanize/humanize.go`](https://github.com/miguelmota/cointop/blob/master/pkg/humanize/humanize.go) we use `LC_NUMERIC` and `LC_MONETARY` for currency formatting. I think we can do something similar for date formatting.

Googling around found this [library](https://github.com/goodsign/monday) that is good for translating date language given locale. E.g.

![DeepinScreenshot_select-area_20210927113213](https://user-images.githubusercontent.com/168240/134965612-6acca1a4-4583-41ff-8929-7b9473359a52.png)
![DeepinScreenshot_select-area_20210927113224](https://user-images.githubusercontent.com/168240/134965613-8bf44318-7d3f-4ebf-b8d3-e1adf8dbc0b7.png)

I don't know anything about i18n it's up for discussion 😃